tools/privacyidea-diag

```diff
@@ -156,51 +180,51 @@ apache_log() {
 		tail -50 /var/log/httpd/ssl_error_log  >> "$tempfile"
 		log
 		log "SECTION: httpd Access_log"
-		log "============================"
+		log "========================="
 		tail -50 /var/log/httpd/ssl_access_log  >> "$tempfile"
+		log
```

Comment:
I think we should include at least the 100 last lines here, just to be on the safe side.